--- hello.c	2015-02-07 14:58:40.252660497 +0000
+++ hallo.c	2015-02-07 15:33:52.892604090 +0000
@@ -1,4 +1,4 @@
-/* hello -- print a greeting message and exit.
+﻿/* hello -- print a greeting message and exit.
    Copyright (C) 1992 Free Software Foundation, Inc.
 
    This program is free software; you can redistribute it and/or modify
@@ -246,7 +246,8 @@
 
   exit (0);
 }
-
+
+
 /* Used by alloca.c.  */
 
 char *
